### PR TITLE
[zh] Resync and fix kube-scheduler reference

### PR DESCRIPTION
**Pull request by tengqm**

### CODE REVIEW

File: content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md [hunk 1]
```diff
@@ -1,1016 +1,1153 @@
...
-  <colgroup>
-    <col span="1" style="width: 10px;" />
-    <col span="1" />
-  </colgroup>
-  <tbody>
+<colgroup>
+<col span="1" style="width: 10px;" />
+<col span="1" />
+</colgroup>
+<tbody>
```
Comment (chenrui333) [inline content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md:43]: interesting, the code is formatted.

**Review by chenrui333 — COMMENTED** [review]
I think there is some HTML code format issues, not sure if we want to sort them out first.

File: content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md (hunk 1 above)
Comment (tengqm) [reply content/zh/docs/reference/command-line-tools-reference/kube-scheduler.md:43]: will check.

**Comment by tengqm** [comment]
Preview: https://deploy-preview-26182--kubernetes-io-master-staging.netlify.app/zh/docs/reference/command-line-tools-reference/kube-scheduler/

**Review by SataQiu — COMMENTED** [review]
/approve
LGTM

**Review by pigletfly — APPROVED** [review]

**Pull request merged**